NuGetPackageVerifier.json

```diff
@@ -6,13 +6,20 @@
         "packages": {
             "Microsoft.VisualStudio.Web.CodeGeneration": { },
             "Microsoft.VisualStudio.Web.CodeGeneration.Core": { },
+            "Microsoft.VisualStudio.Web.CodeGeneration.Design": { },
             "Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore": { },
             "Microsoft.VisualStudio.Web.CodeGeneration.Templating": { },
             "Microsoft.VisualStudio.Web.CodeGeneration.Tools": { },
```

Comment:
Are planning to add 'packageType = DotnetCliTool' soon? cref https://github.com/aspnet/Coherence-Signed/issues/367 and https://github.com/aspnet/DotNetTools/blob/f8a1a66ab7962c2030528d68aabb94a41c5ca3e8/NuGetPackageVerifier.json#L7-L14